scripts/updateVersion.mjs

```diff
@@ -0,0 +1,128 @@
+import semver from "semver";
+import { exec } from "child_process";
+import { readFile, writeFile } from "fs/promises";
+
+const REPO_URL = "https://github.com/nrkno/sofie-core";
+const START_OF_LAST_RELEASE_PATTERN =
+	/(^#+ \[?[0-9]+\.[0-9]+\.[0-9]+|<a name=)/m;
+const HEADER = `# Changelog\n\nAll notable changes to this project will be documented in this file. See [Convential Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification) for commit guidelines.\n\n`;
+
+const execPromise = (command) =>
```

Comment:
I would recommend https://www.npmjs.com/package/zx to simplify running of stuff like this
I know Jan raised some concern about windows compatability when I suggested it before, but I havent yet had any complaints from companion after I rewrote a bunch of scripts to zx.

Reply:
I've not used zx for this because it requires a minimum node version of 16